tests/manage/pv_services/test_reclaim_policy.py

```diff
@@ -67,7 +67,7 @@ class TestReclaimPolicy(ManageTest):
     """
 
     @pytest.mark.polarion_id("OCS-383")
-    def test_reclaim_policy_retain(self):
+    def reclaim_policy_retain(self):
```

Comment:
might be worth deleting than if its already duplicated ?

Reply:
I wasn't sure if deletion is preferred so just disabled them.
We can delete if that is an agreement for that.
@ebenahar @keesturam

Reply:
@sidhant-agrawal @ebenahar any update here? Deletion is preferred for *dead* code, git still shows the history if *at all* needed, lets keep the code that is in use.

Reply:
Now removing the `test_reclaim_policy.py`